packages/ui/src/components/molecules/inputs/DatePickerInput/DatePickerInput.tsx

```diff
@@ -69,8 +76,8 @@ export const DatePickerInput = ({
     return deserializeValue(_value);
   }, [_value, deserializeValue]);
 
-  const Field = useMemo(
-    () => (props: TextFieldProps) => {
+  const Field = useMemo(() => {
```

Comment:
Why not pass `TextField` directly or use the `memo` function?

Reply:
Non memoized TextField will generate bug where user is going to lose focus on each keystroke. `memo` cant be used because this field is accessing internal component props and some of them are cached by useMemo as their initial version.